tests/dev/script-create-individual.R

```diff
@@ -0,0 +1,30 @@
+#Create Individual example.
+#Assuming PKSim installed
+
+initPKSim("C:/projects/PK-Sim/src/PKSim.R/bin/Debug/net472")
+
+
+originData <- OriginData$new()
+originData$species <- Species$Human
+originData$population <- HumanPopulation$European_ICRP_2002
+originData$gender <- Gender$Female
+
+print(originData)
+
+originData <- createOriginData(

```

Comment:
Example of usage

Reply:
@Yuri05 @PavelBal The more I think about it, the more I think we should hide the origin data from the user API

We could expose the `IndividualCharacteristics` object that I had to create in PKSim thast has the originData and the MoleculeOntogeny. That way it can be reused etc.. 
Or we just have a big long list of parameters as we haev in Matlab. Not so nice but why not

Reply:
Yes I think it is a good idea to expose `IndividualCharacteristics`.

I don't like the matlab-approach of having these lists of parameters, better to summarize them in an object.